### PR TITLE
#3869 align the dependencies with kafka 2.4.0

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3869)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-configurator/src/test/scala/com/island/ohara/configurator/TestConfiguratorBuilder.scala [hunk 1]
```diff
@@ -29,8 +29,7 @@ import com.island.ohara.common.util.{CommonUtils, Releasable, VersionUtils}
 import com.island.ohara.configurator.Configurator.Mode
 import org.junit.Test
 import org.scalatest.Matchers._
-import org.scalatest.mockito.MockitoSugar
-import org.scalatest.mockito.MockitoSugar._
+import org.scalatestplus.mockito.MockitoSugar
```
Comment (vitojeng) [inline ohara-configurator/src/test/scala/com/island/ohara/configurator/TestConfiguratorBuilder.scala]: Why use  **org.scalatestplus.mockito.MockitoSugar** instead **org.scalatest.mockito.MockitoSugar** ?

Comment (chia7712) [reply ohara-configurator/src/test/scala/com/island/ohara/configurator/TestConfiguratorBuilder.scala]: The later is deprecated, and the other is replacement

Comment (vitojeng) [reply ohara-configurator/src/test/scala/com/island/ohara/configurator/TestConfiguratorBuilder.scala]: Got it

**Comment by chia7712** [comment]
retry shabondi

**Comment by chia7712** [comment]
retry kafka

**Comment by jackyoh** [comment]
retry kafka

**Comment by vitojeng** [comment]
I guess I should fix the flaky test failure in **com.island.ohara.shabondi.TestRoute.testSourceRoute**
Will file a issue for this.

**Comment by chia7712** [comment]
flaky is traced by #3867

**Pull request merged**